.github/workflows/publish.yml

```
@@ -50,9 +50,7 @@ jobs:
         run: pnpm build
 
       - name: Publish package
-        run: pnpm run ci-publish ${{ github.ref_name }}
-        env:
-          NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
+        run: npm i -g npm@latest && pnpm run ci-publish ${{ github.ref_name }}
```

```suggestion
        run: npm i -g npm@^11.5.2 && pnpm run ci-publish ${{ github.ref_name }}
```
Let's specify the major version so that a major bump is not introduced without noticing.